CHANGELOG.md

```diff
@@ -1,15 +1,51 @@
 <!--
-# v0.12.0
+# v0.13.0
 _2018_
+  - [Downloads for v0.13.0](https://github.com/okkur/syna/releases/tag/v0.13.0)
+  - [Changelog since v0.12.0](#changes-since-v012)
+
+## Documentation for v0.13.0
+[Documentation](/tree/v0.13.0/docs)
+
+## Changes since v0.12.0
+
+## Fixes since v0.12.0
+
+---
+
+-->
+
+# v0.12.0
+_2018-08-06
   - [Downloads for v0.12.0](https://github.com/okkur/syna/releases/tag/v0.12.0)
   - [Changelog since v0.11.0](#changes-since-v011)
 
 ## Documentation for v0.12.0
-[Documentation](/tree/v0.11.0/docs)
+[Documentation](/tree/v0.12.0/docs)
+[Getting started](/tree/v0.12.0/docs#using-starter)
 
 ## Changes since v0.11.0
+  - BREAKING: `content-single` and `content-split` merged into `content` fragment
+  - BREAKING: Moving to `_index` and `_global` as special directories and headless bundles
+  - BREAKING: Subpath handling made consistent with Hugo
+  - New: `header` fragment for easier section bundling and linking
```

Comment:
`header` -> `section-header`

Should I rename it to `header`?

Reply:
Mhh might make sense to just go with header?

Reply:
Added a migration commit. Any arguments against going with the simpler one aka `header`?

Reply:
I think we had one but don't remember the outcome. Header sounds good.